mlbench_core/evaluation/pytorch/inference/__init__.py

```diff
@@ -0,0 +1 @@
+from .translator import Translator
```

Comment:
Maybe the classes in the `inference` namespace would better fit in the `mlbench_core.utils` namespace or alongside the model. the `Evaluation` namespace is meant for losses and measures like accuracy, f1 score etc.